entrypoint_debian.sh

```diff
@@ -52,14 +33,52 @@ case ${OPENHAB_VERSION} in
         cp -av "${APPDIR}/configurations.dist/." "${APPDIR}/configurations/"
       fi
     ;;
-  2.0.0|2.1.0|2.2.0|2.3.0-snapshot)
+  2.0.0|2.1.0|2.2.0-snapshot)
```

Comment:
Looks like some changes were not properly merged since this changes the line back to a previous version.

Reply:
:-| I literally cloned the repo a few hours ago. This is weird.